### ISSUE TITLE
File type in Resume previous alignment 

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alignment-editor-new#450]
Reported in issue #411 
Based on the file type, it is not possible to understand if it's a full alignment json file or not.
see Resume previous alignment list from splash screen:

![Screen Shot 2021-07-05 at 5 19 14 PM](https://user-images.githubusercontent.com/41396793/124492763-2f75cb00-ddb5-11eb-9548-2a406ace5b97.png)

file fra_eng is a partial alignment but it's not obvious based on the file type in Resume previous alignment in splash screen. This might be out of scope but I think it's worth to report this.

**Comment by irina060981** [comment]
There is no file type here - these are records from database.
It shows only that data - that is saved to this table that is used for showing alignment lists (to avaoid loading to much unuseful information).

I have added the column with the information whether alignment is tokenized or not. But it is not updated automatically - it would be updated only after alignment would be opened once at least.

ALso I added default sorting - by date updated in reverse.

![image](https://user-images.githubusercontent.com/12377640/124582590-7c30d300-de95-11eb-819d-0b5e50ca577e.png)

**Comment by monzug** [comment]
Thanks for ordering the files.

**Comment by irina060981** [comment]
>Thanks for ordering the files. can we change 'has tokens' to 'with tokens' or just 'tokens'?

Yes, certainly.
What should I use - `no tokens` and `with tokens`?

**Comment by monzug** [comment]
I think so. Thanks

**Comment by irina060981** [comment]
Fixed
